en/plugins.rst

```diff
@@ -646,6 +646,14 @@ name would be `foo-bar/cakephp-logging`.
 And the CakePHP owned "Localized" plugin can be found under `cakephp/localized`
 respectively.
 
+Manage Your Plugins using Mixer
+===================
```

Comment:
Title underlines need to be the same length as the text above them.

Reply:
Updated.